Sourabh_Kanwade/html/Assignment 6/question_2.css

```diff
@@ -0,0 +1,57 @@
+body {
+    font-family: Arial, Helvetica, sans-serif;
+}
+h1,
+h2 {
+    background-color: blueviolet;
+    border-bottom-left-radius: 10px;
+    border-top-right-radius: 10px;
+    display: flex;
+    justify-content: flex-end;
+    padding: 5px;
+}
+h1 {
+    margin-bottom: 50px;
+}
+label {
+    font-weight: bold;
+}
+#first-name {
+    margin: 20px 0;
+}
+#last-name {
+    margin: 20px 0;
+}
+#other {
+    margin: 20px 0;
+}
+.container {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 70%;
+}
+
```

Comment:
remove space from here and push changes as soon as possible

Reply:
done